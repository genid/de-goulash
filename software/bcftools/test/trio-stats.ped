HG00100 child1 father1 mother1 2 0 CEU
HG00101 father1 0 0 1 0 CEU
HG00102 mother1 0 0 2 0 CEU
HG00200 child2 father2 mother2 2 0 CEU
HG00201 father2 0 0 1 0 CEU
HG00202 mother2 0 0 2 0 CEU
