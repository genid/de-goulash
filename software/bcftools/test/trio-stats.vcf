##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##contig=<ID=20,length=81195210>
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	child1	father1	mother1	child2	father2	mother2
20	302	.	T	TA	.	.	.	GT	0/1	0/0	1/1	0/1	1/0	1/1
20	303	.	T	C	.	.	.	GT	0/1	1/1	1/1	0/1	1/0	1/1
20	304	.	T	A	.	.	.	GT	0/0	0/1	1/1	0/1	1/0	1/1
20	305	.	T	A	.	.	.	GT	1/1	0/0	0/0	0/0	0/0	0/0
20	306	.	T	A	.	.	.	GT	0/0	0/1	0/0	0/0	0/0	0/0
20	307	.	T	A	.	.	.	GT	0/1	1/0	0/1	0/1	1/0	0/0
20	308	.	T	A,C	.	.	.	GT	0/0	0/0	0/0	0/1	1/2	0/0
20	309	.	T	A,C	.	.	.	GT	1/2	1/1	1/1	1/1	1/1	1/1
20	310	.	T	A,C	.	.	.	GT	1/1	1/2	1/1	1/1	1/1	1/1
20	311	.	T	A,C	.	.	.	GT	1/1	1/1	1/1	1/2	2/1	1/1
20	312	.	TAA	T	.	.	.	GT	0/1	1/0	0/0	0/0	0/0	0/0
20	313	.	A	C,*	.	.	.	GT	0/2	2/0	0/0	0/0	1/0	1/0
20	314	.	A	C,*	.	.	.	GT	1/2	2/0	1/0	0/0	1/0	1/0
20	315	.	A	C	.	.	.	GT	1/0	1/1	1/0	1/1	1/0	1/1
