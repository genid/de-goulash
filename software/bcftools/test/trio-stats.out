# CMD line shows the command line used to generate this output
# DEF lines define expressions for all tested thresholds
# FLT* lines report numbers for every threshold and every trio:
#   1) filter id
#   2) child
#   3) father
#   4) mother
#   5) number of valid trio genotypes (all trio members pass filters, all non-missing)
#   6) number of non-reference trio GTs (at least one trio member carries an alternate allele)
#   7) number of DNMs/Mendelian errors
#   8) number of novel singleton alleles in the child (counted also as DNM / Mendelian error)
#   9) number of untransmitted trio singletons (one alternate allele present in one parent)
#   10) number of transmitted trio singletons (one alternate allele present in one parent and the child)
#   11) number of transitions, all distinct ALT alleles present in the trio are considered
#   12) number of transversions, all distinct ALT alleles present in the trio are considered
#   13) overall ts/tv, all distinct ALT alleles present in the trio are considered
#   14) number of homozygous DNMs/Mendelian errors (likely genotyping errors)
#   15) number of recurrent DNMs/Mendelian errors (non-inherited alleles present in other samples; counts GTs, not sites)
MERR	20	303	child1	father1	mother1	-	RECURRENT
MERR	20	304	child1	father1	mother1	HOM	RECURRENT
MERR	20	305	child1	father1	mother1	HOM	-
TRANSMITTED	20	306	child1	father1	mother1	NO
TRANSMITTED	20	308	child2	father2	mother2	YES
TRANSMITTED	20	308	child2	father2	mother2	NO
MERR	20	309	child1	father1	mother1	-	-
TRANSMITTED	20	310	child1	father1	mother1	NO
TRANSMITTED	20	311	child2	father2	mother2	YES
TRANSMITTED	20	312	child1	father1	mother1	YES
DEF	FLT0	all
FLT0	child1	father1	mother1	14	12	4	2	2	1	3	9	0.33	2	2
FLT0	child2	father2	mother2	14	11	0	0	1	2	3	9	0.33	0	0
